src/tracking/index.js

```diff
@@ -0,0 +1,36 @@
+import ReactGA from "react-ga";
+
+export const initGA = (trackingID) => {
+  ReactGA.initialize(trackingID, { testMode: process.env.NODE_ENV === "test" });
+};
+
+export const PageView = () => {
+  ReactGA.pageview(window.location.pathname + window.location.search);
+};
+
+/**
+ * Event - Add custom tracking event.
+ * @param {string} category
+ * @param {string} action
+ * @param {string} label
+ */
+export const Event = (category, action, label) => {
+  ReactGA.event({
+    category: category,
+    action: action,
+    label: label,
+  });
+};
+
+/**
+ * OutboundLink - Track clicks that are not page links.
+ * @param {string} label
+ * @param {string} to
+ */
+export const OutboundLink = (label, to) => {
+  ReactGA.OutboundLink({
+    eventLabel: label,
+    to: to,
+    target: "_self",
+  });
+};
```

Comment:
Cool, so this is what we'll use for any external links, right?

Reply:
I have not tested this, the code could be incorrect here but this was in the example of the react-ga npm documentation. The theory here is that we can generate an event on things that are not links such as a logo to gain better insights to how the users are interacting with the page.

Reply:
oOo, didn't realize we could implement it for other elements, too.  Nice way to track data-set usage.